plugins/woocommerce/bin/build-zip.sh

```diff
@@ -21,15 +24,15 @@ if [ -z "${NODE_ENV}" ]; then
 fi
 pnpm --filter='@woocommerce/plugin-woocommerce' build || exit "$?"
 echo "Cleaning up PHP dependencies..."
-composer install --no-dev || exit "$?"
+composer install --no-dev --quiet --optimize-autoloader || exit "$?"
```

Comment:
I don't recall seeing it tried, but it's worth giving it a shot.  Thanks for this.

Reply:
Gotcha! @woocommerce/vortex: in case we discover any autoloading issues after merging this PR, it could be due to `--optimize-autoloader` flag here.

The flag aims to improve autoloading times a bit, and the Jetpack autoloader readme mentions it as well, so let's see what happens.